### DISCUSSION TITLE
Air Purifier not showing up

### DISCUSSION THREAD

**Discussion by Birdheezy** [SecKatie/ha-wyzeapi#547] [Q&A]
Hello and thanks for this integration! 
All of my bulbs and outlets show up just fine, but I have the wyze air purifier and it's now showing up in home assistant. I've added and removed this integration twice hoping I messed something up, but I'm still not seeing it. 
Thanks again for all the work on this integration!

**Comment by shomanjk** [comment]
I don't see it either, but it would be great to have, especially if it also included the ability to pull the air quality sensor data too.
